### ISSUE TITLE
support deploy-mgmt-hub.sh for CentOS

### ISSUE THREAD

**Issue by janvda** [open-horizon/devops#31]
Currently as is described in the [README.md](https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md): the script `deploy-mgmt-hub.sh` is only supported for Ubuntu 18.x and macOS.
It would be nice to extend the support to CentOS so that I can deploy the all-in-1 Horizon Management Hub, Agent and CLI on my Intel NUC6CAYB running CentOS Stream 8.

**Comment by bmpotter** [comment]
@janvda interested in contributing a PR for this?

**Comment by janvda** [comment]
I can give it a shot and see how far I get.

**Comment by janvda** [comment]
I had a quick attempt.

There are 2 blocking things:
1.  ubuntu is using different commands (`apt-get`) to install packages than CentOS (rpm)
2. The script is installing the horizon debian package (`apt-get install -yqf $TMP_DIR/pkgs/horizon*.deb`).  So we need an `rpm` version of the horizon package for CentOS.

**Comment by bmpotter** [comment]
@janvda See https://github.com/open-horizon/anax/blob/master/agent-install/agent-install.sh for examples of the rpm commands

**Comment by Anitaachu** [comment]
@bmpotter @joewxboy I would love to take a swing at this issue

**Comment by joewxboy** [comment]
@janvda Are you still working on this issue?  If not, can @Anitaachu give it a shot?

**Comment by janvda** [comment]
No, I am not working on it. @Anitaachu can certainly give it a shot.

**Comment by joewxboy** [comment]
OK, @Anitaachu let @bmpotter know if you have any questions.

**Comment by naphelps** [comment]
With RHEL ppc64le and Fedora 36+ support, this should be simple to add.
